src/analyses/apron/apronPriv.apron.ml

```diff
@@ -503,10 +505,10 @@ sig
 end
 
 (** No clustering. *)
-module NoCluster: ClusterArg =
+module NoCluster:ClusterArg = functor (AD: ApronDomain.S2) ->
 struct
-  open CommonPerMutex
-
+  module AD = AD
```

Comment:
These `module AD = AD` lines are unnecessary, no?

Reply:
Yes, my bad. They were necessary in some previous state when the `AD` needed to be re-exported. 
I can directly change that on `master`, if that's ok?

Reply:
If they're there too, then sure. I thought it was somehow introduced in the merge conflicts, like the commented out code.